ergo-wallet/src/main/scala/org/ergoplatform/wallet/transactions/TransactionBuilder.scala

```diff
@@ -92,10 +93,15 @@ object TransactionBuilder {
     val tokensOutNoMinted = tokensOut.filterKeys(_ != firstInputBoxId)
     val mintedTokensNum = tokensOut.size - tokensOutNoMinted.size
     require(mintedTokensNum <= 1, s"Only one token can be minted, but found $mintedTokensNum")
+    require(burnTokens.values.forall(_ > 0), s"Incorrect burnTokens specification: $burnTokens")
```

Comment:
While other checks are protocol specific, this particular is not, Please make a comment about that.

Reply:
done